source/app/service-providers/fsal/util/search-file.ts

```diff
@@ -72,7 +72,7 @@ export default function searchFile (fileObject: MDFileDescriptor|CodeFileDescrip
           // Break because only one match necessary
           break
         }
-      } else if (wd[0] === '#' && fileObject.type === 'file' && fileObject.tags.includes(wd.toLowerCase().substring(1))) {
+      } else if (wd[0] === '#' && fileObject.type === 'file' && fileObject.tags.map((each) => each.toLowerCase()).includes(wd.toLowerCase().substring(1))) {
```

Comment:
You should absolutely do the lowercasing in the file parser above, and NOT here! Always normalize the tags where they are at.